### PR TITLE
Update 2019-12-python.md

### PR DESCRIPTION
**Pull request by chlowell**

### CODE REVIEW

File: releases/2019-12/2019-12-python.md [hunk 1]
```diff
@@ -43,11 +43,19 @@ If you have a bug or feature request for one of the libraries, please post an is
 
 Detailed change logs are linked to in the Quick Links below. Here are some critical call outs.
 
+### Identity
+
+- `InteractiveBrowserCredential` raises when unable to open a web browser
+- The credentials composing `DefaultAzureCredential` are configurable by keyword
+arguments
+- `SharedTokenCacheCredential` accepts an optional `tenant_id` keyword argument
```
Comment (adrianhall) [inline releases/2019-12/2019-12-python.md]: Do these really rise to the level of a key feature or improvement?  Are these critical?  I agree they should be in the CHANGELOG, but I don't see them rising to the level of "critical"

Comment (chlowell) [reply releases/2019-12/2019-12-python.md]: That's reasonable. Then the note is just e.g. "released version 1.1.0"?

Comment (adrianhall) [reply releases/2019-12/2019-12-python.md]: Something like "Released version 1.1.0 of the azure-identity library, which fixes customer reported and usability issues"

**Pull request merged**